### PR TITLE
Enable tailwind prettier plugin

### LINKED ISSUE #29: Add prettier tailwind plugin
https://github.com/tailwindlabs/prettier-plugin-tailwindcss

### PR DESCRIPTION
**Pull request by jsun969**
### Description
Enable tailwind prettier plugin

### Changes Made
1. Enable tailwind prettier plugin
2. Add type for `prettierrc`
3. Avoid formatting `pnpm-lock.yaml` (remove `.yaml` in format command)
4. Re-format

### Related Issues
close #29

### CODE REVIEW

File: package.json
Comment (rayokamoto) [inline package.json:1]: I think we should still have `.yaml` as some configs might require `.yaml` over `.yml` and both are valid file extensions

**Review by rayokamoto — CHANGES_REQUESTED** [review]

File: package.json
Comment (rayokamoto) [reply package.json:1]: Is there a way to just ignore pnpm-lock.yaml?

**Review by rayokamoto — APPROVED** [review]
Oh of course, I forgot the `.prettierignore` existed. Happy to have this merged now.

**Comment by jsun969** [comment]
> Oh of course, I forgot the `.prettierignore` existed. Happy to have this merged now.

TBH, I forgot that either XDDDD

**Pull request merged**